zetta_utils/augmentations/imgaug.py

```diff
@@ -58,21 +58,14 @@ def _ensure_cxyn(x: Sequence[Tensor], ref: Sequence[TensorTypeVar]) -> list[Tens
     ...
 
 
-def _ensure_cxyn(
-    x: Tensor | Sequence[Tensor], ref: TensorTypeVar | Sequence[TensorTypeVar]
-) -> TensorTypeVar | list[TensorTypeVar]:
+def _ensure_cxyn(x, ref):
```

Comment:
Oh, I should only type the @overload versions?

Reply:
yeah, that's the proper behavior as per official docs [https://docs.python.org/3/library/typing.html]
although sometimes VSCode encourages incorrect behavior by working better with explicit annotation
![Uploading image.png…]()